lms/models/application_instance.py

```diff
@@ -38,6 +41,24 @@ class ApplicationInstance(BASE):
         nullable=False,
     )
 
+    # LMS info linked to this consumer_key, description of each one from the spec
+    # https://www.imsglobal.org/specs/ltiv1p1p1/implementation-guide
+
+    # This is a unique identifier for the TC.  A common practice is to use the DNS of the organization or the DNS of the TC instance.
+    tool_consumer_instance_guid = sa.Column(sa.String, nullable=True)
```

Comment:
I don't think this is unique, it's "immutable" but we enforce that in the code

Reply:
Yeah, not unique